build.gradle

```diff
@@ -35,45 +46,76 @@ dependencyManagement {
+                details.useVersion("4.6.1")
+            }
+        }
+    }
 }
 
 dependencies {
-    api "org.liquibase:liquibase-core:$liquibaseVersion"
+    api "org.liquibase:liquibase-core:$liquibaseHibernate5Version"
 
-    api("org.liquibase.ext:liquibase-hibernate5:$liquibaseHibernateVersion") {
+    api("org.liquibase.ext:liquibase-hibernate5:$liquibaseHibernate5Version") {
         exclude group: 'org.hibernate', module: 'hibernate-core'
         exclude group: 'org.hibernate', module: 'hibernate-entitymanager'
         exclude group: 'org.hibernate', module: 'hibernate-envers'
         exclude group: 'org.liquibase', module: 'liquibase-core'
         exclude group: 'org.liquibase', module: 'liquibase-test-harness'
         exclude group: 'com.h2database', module: 'h2'
     }
-    compileOnly "org.hibernate:hibernate-core:5.6.15.Final"
+    api "jakarta.annotation:jakarta.annotation-api:$jakartaAnnotationApiVersion"
+    api 'org.apache.commons:commons-lang3:3.16.0'
+
+    compileOnly "org.hibernate:hibernate-core-jakarta:$hibernate5Version"
 
-    compileOnly 'org.springframework.boot:spring-boot-starter-logging'
+    compileOnly "org.springframework.boot:spring-boot-starter-logging"
     compileOnly "org.springframework.boot:spring-boot-starter-actuator"
     compileOnly "org.springframework.boot:spring-boot-autoconfigure"
     compileOnly "org.springframework.boot:spring-boot-starter-tomcat"
 
     compileOnly "org.grails:grails-web-boot"
     compileOnly "org.grails:grails-dependencies"
-    compileOnly 'javax.servlet:javax.servlet-api:4.0.1'
-    compileOnly "org.grails.plugins:hibernate5"
+    compileOnly "jakarta.servlet:jakarta.servlet-api:$servletApiVersion"
+    compileOnly "org.grails.plugins:hibernate5:$gormVersion"
+    testImplementation "io.micronaut:micronaut-inject-groovy:$micronautVersion"
+    testImplementation "io.micronaut:micronaut-inject:$micronautVersion"
+    testImplementation "org.liquibase.ext:liquibase-hibernate5:$liquibaseHibernate5Version"
+    testImplementation "org.liquibase:liquibase-core:$liquibaseHibernate5Version"
+
 
     testImplementation "org.springframework.boot:spring-boot-starter-tomcat"
-    testImplementation "org.hibernate:hibernate-core:5.6.15.Final"
-    testImplementation "org.grails.plugins:hibernate5"
+    testImplementation "org.hibernate:hibernate-core-jakarta:$hibernate5Version"
+    testImplementation "org.grails.plugins:hibernate5:$gormVersion"
 
     testImplementation "org.grails:grails-gorm-testing-support"
     testImplementation "org.mockito:mockito-core"
     testImplementation "org.grails:grails-web-testing-support"
-    testImplementation group: 'com.h2database', name: 'h2', version: '1.4.200'
+    testImplementation "com.h2database:h2:$h2Version"
+
 
+    documentation "org.apache.groovy:groovy:$groovyVersion"
+    documentation "org.apache.groovy:groovy-ant:$groovyVersion"
+    documentation "org.apache.groovy:groovy-templates:$groovyVersion"
```

Comment:
I think we weed to add `documentation "com.github.javaparser:javaparser-core:$javaParserCoreVersion"` here for the groovydoc task to work. (And for example the version `3.25.10` in `gradle.properties`)

Reply:
Good catch, I have added it.